docs/releases/5.1.1.txt

```diff
@@ -12,3 +12,8 @@ Bugfixes
 * Fixed a regression in Django 5.1 that caused a crash of ``Window()`` when
   passing an empty sequence to the ``order_by`` parameter, and a crash of
   ``Prefetch()`` for a sliced queryset without ordering (:ticket:`35665`).
+
+* Fixed a regression in Django 5.1 where a new ``usable_password`` field
+  was included in the :class:`~django.contrib.auth.forms.BaseUserCreationForm`,
+  affecting its extensibility and causing unintended side effects when
+  subclassing this form for user creation customization (:ticket:`35678`).
```

Comment:
We might want to mention the new ```:class:`~django.contrib.auth.forms.AdminUserCreationForm` ``` here and maybe that `UserCreationForm` was affected (if being used outside the admin context) but not sure on how best to word it

Reply:
Thanks, I will work something out!